Example/package.json

```diff
@@ -1,21 +1,21 @@
 {
   "name": "Example",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.0.0-alpha.12",
-    "react-native": "0.46.0",
+    "react": "16.0.0",
```

Comment:
Are you sure this is correct? Because react-native `0.49.3` still references the beta; `"react": "16.0.0-beta.5"`

Reply:
good catch!